tests/vm_migration_test.go

```diff
@@ -93,30 +91,26 @@ var _ = Describe("VmMigration", func() {
 
 			time.Sleep(time.Second * 3)
 
-			r, err = restClient.Get().Resource("vms").Namespace(api.NamespaceDefault).Name(vm.ObjectMeta.Name).Do().Get()
-			var vmR *v1.VM = r.(*v1.VM)
-			Expect(vmR.ObjectMeta.Name).To(Equal(vm.ObjectMeta.Name))
+			Eventually(func() string {
```

Comment:
You probably have to give your Eventually calls an overall timeout and a polling intervall different to 1 second and 10 ms. Or did it just work for you?